src/ui/Maps.css

```diff
@@ -0,0 +1,28 @@
+.widget-leaflet-maps-wrapper {
```

Comment:
Are you combining styles from different widgets? This is so not a maintainable way of doing this... Split them.